### DISCUSSION TITLE
Integrate with a custom plugin

### DISCUSSION THREAD

**Discussion by agenceKanvas** [Pierre-Lannoy/wp-decalog#59] [Q&A] [answered]
Hi ! 
So, I'm trying to log the events of my plugin with Decalog. 
![Capture d’écran 2024-03-25 à 16 27 48](https://github.com/Pierre-Lannoy/wp-decalog/assets/190149/8d0779a8-2fa8-4f90-9e22-0ed1b1bb7b5c)
The autosubscribe has worked (it appears in the list). 

I've added in my admin_enqueue_scripts function some tests :
```
$events  = \DecaLog\Engine::eventsLogger( 'kvb' );
	$events->info( 'This is an information.' );
	$events->error( 'This is an error with a code.', [ 'code' => 500 ] );
```
But I'm missing something, I don't get how I can see them in the admin. Or anywhere else :) 
Any pointer please ?

Thanks !

**Comment by Pierre-Lannoy** [comment]
Hello !
To "view" these events you have to create loggers ("journaliseurs" in French). These loggers can be local (i.e. in WordPress DB or APCu) or external services. To start understanding how DecaLog works, you can first add a WordPress event logger and see what's coming ;)

**Comment by agenceKanvas** [comment]
Ok thanks ! I will do that. 
Strange thing, I had to change the method : 

```
$logger = \DecaLog\Engine::getLogger( 'kvt' );
$logger->info( 'Test' );
```

To 

```
$logger = \DecaLog\Engine::eventsLogger( 'kvt' );
$logger->info( 'Test' );
```

Maybe I did something wrong in the init but at least right onw I have the test appearing. Thanks !

**Reply by Pierre-Lannoy** [reply, marked as answer]
I confirm the second way to call DecaLog is the right one ;) 
https://decalog.io/sdk/loggin-events/
